rosdep/base.yaml

```
@@ -3166,6 +3166,10 @@ libespeak-dev:
   gentoo: [app-accessibility/espeak]
   nixos: [espeak]
   ubuntu: [libespeak-dev]
+libespeak-ng1:
+  debian: [libespeak-ng1]
+  opensuse: [libespeak-ng1]
+  ubuntu: [libespeak-ng1]
```

Please add rules for Fedora and RHEL as well.

Fedora: https://packages.fedoraproject.org/pkgs/espeak-ng/espeak-ng/
RHEL: https://repo.almalinux.org/almalinux/8/AppStream/x86_64/os/Packages/espeak-ng-1.49.2-4.el8.i686.rpm

```suggestion
libespeak-ng1:
  debian: [libespeak-ng1]
  fedora: [espeak-ng]
  opensuse: [libespeak-ng1]
  rhel:
    '*': [espeak-ng]
    '7': null
  ubuntu: [libespeak-ng1]
```